website/docs/d/vapp.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp"
+sidebar_current: "docs-vcd-datasource-vapp"
+description: |-
+  Provides a vCloud Director vApp data source. This can be used to reference vApps.
+---
+
+# vcd\_vapp
+
+Provides a vCloud Director vApp data source. This can be used to reference vApps.
+
+Supported in provider *v2.5+*
+
+## Example Usage
+
+
+```hcl
+data "vcd_vapp" "test-tf" {
+  name             = "test-tf"
+  org              = "tf"
+  vdc              = "vdc-tf"
+}
+
+output "name" {
+  value = data.vcd_vapp.test-tf.name
+}
+
+output "description" {
+  value = data.vcd_vapp.test-tf.description
+}
+
+output "href" {
+  value = data.vcd_vapp.test-tf.href
+}
+
+output "status_text" {
+  value = data.vcd_vapp.test-tf.status_text
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A unique name for the vApp
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
+
+## Attribute reference
```

Comment:
If everyone knows that `Attributes` are `Computed` ? - just thinking if should that mentioned

Reply:
Everything except the mandatory items is computed in data source

Reply:
For new users isn't obvious, I would say